src/core/modules/Orbit.hpp

```diff
@@ -167,7 +165,7 @@ class GimbalOrbit : public Orbit {
 	void setLatitude(const double lat) { latitude=lat*M_PI_180;}
 	void setDistance(const double dist){ distance=dist;}
 	void addToLongitude(const double dlong){ longitude+=dlong*M_PI_180; }
-	void addToLatitude(const double dlat)  { latitude=qBound(-M_PI_2, latitude+dlat*M_PI_180, M_PI_2);}
+	void addToLatitude(const double dlat)  { latitude=qBound(-89.99*M_PI_180, latitude+dlat*M_PI_180, 89.99*M_PI_180);}
```

Comment:
This is ugly. You'll see this if you try to move to the pole from SSO, looking at the Sun. My attempt came closer to the desired result (basically the same, but with closer value to 90°):

```diff
diff --git a/src/core/modules/Orbit.cpp b/src/core/modules/Orbit.cpp
index b352e53c52..99bdfcebca 100644
--- a/src/core/modules/Orbit.cpp
+++ b/src/core/modules/Orbit.cpp
@@ -293,6 +293,16 @@ GimbalOrbit::GimbalOrbit(double distance, double longitude, double latitude):
 {
     setParentOrientation(0., 0., 0.);
 };
+
+void GimbalOrbit::addToLatitude(const double dlat)
+{
+    // This value lets us avoid jumpiness of view roll when looking at the poles
+    const auto epsilon = 1e-8;
+
+    const auto max = M_PI_2*(1 - epsilon);
+    latitude = qBound(-max, latitude+dlat*M_PI_180, +max);
+}
+
 //! Compute the position (JDE is just a placeholder) and return a "stellarium compliant" position
 void GimbalOrbit::positionAtTimevInVSOP87Coordinates(double JDE, double* v)
 {
diff --git a/src/core/modules/Orbit.hpp b/src/core/modules/Orbit.hpp
index 64e54a1544..71c12a0ad2 100644
--- a/src/core/modules/Orbit.hpp
+++ b/src/core/modules/Orbit.hpp
@@ -167,7 +167,7 @@ public:
     void setLatitude(const double lat) { latitude=lat*M_PI_180;}
     void setDistance(const double dist){ distance=dist;}
     void addToLongitude(const double dlong){ longitude+=dlong*M_PI_180; }
-    void addToLatitude(const double dlat)  { latitude=qBound(-M_PI_2, latitude+dlat*M_PI_180, M_PI_2);}
+    void addToLatitude(double dlat);
     void addToDistance(const double ddist) { distance=qBound(0.01, distance+ddist, 50.);}
 
 private:
```

Reply:
I attempted avoiding the weird flip when going away from 90°, but neither my 0.1° nor your going even closer solves the problem. Not sure what, but something causes the view to rotate by tens of degrees between 85 and ~90°. Go to the maximum latitude now allowed and rotate (Alt-right/left). See also e.g. for Saturn Observer. Maybe the parent orientation for GimbalOrbit needs more thought. Just that previously the odd motion did not even allow thinking about these details!